HCCGo/app/html/jobView.html

```diff
@@ -21,9 +21,12 @@ <h3 class="text-center">{{job.jobName}} - {{job.jobId}}</h3>
            <div class="panel-heading jobstatus-link" data-toggle="collapse" data-target="#collapse0">
              <div class="row">
                <div class="col-sm-12">
-                 <h4 class="panel-title jobhistory">Output<small class="jobdate"> {{job.outputPath}} </small></h4>
+                 <h4 class="panel-title jobhistory" style="display: inline-block">Output<small class="jobdate"> {{job.outputPath}}</small></h4>
+                 <button type="submit" class="btn btn-primary btn-xs save-buttons" ng-click="saveFile('outText', $event)"> <i class="glyphicon glyphicon-floppy-disk" aria-hidden="True"></i> Save</button>
+                 <button type="submit" class="btn btn-primary btn-xs save-buttons" ng-click="copyToClipboard('outText', $event)"> <i class="glyphicon glyphicon-copy" aria-hidden="true"></i> Copy</button>  
```

Comment:
Doesn't the output only display a certain amount of bytes so they should only be able to copy that right?